src/openrct2/platform/Platform.Linux.cpp

```diff
@@ -56,19 +55,27 @@ namespace OpenRCT2::Platform
         // GNU recommends scripts/programs set LC_ALL to override
         // locales for uniform testing, clearing it after should let
         // LANG and other locale settings operate normally
-        static inline const std::string _kLangVarName{ "LC_ALL" };
-        static inline const std::string _kTargetLang{ "C.UTF-8" };
+        static inline const std::string _kOverrideVarName{ "LC_ALL" };
+        static inline const std::string _kTargetLocale{ "C.UTF-8" };
```

Comment:
Fwiw, these are never used as std::strings (you always invoke `.c_str()`, so let's just make them actual constants:
```suggestion
        static constexpr const char* kOverrideVarName = "LC_ALL";
        static constexpr const char* kTargetLocale = "C.UTF-8";
```